test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/CS/Source/NullConditionalExpressions.cshtml

```diff
@@ -0,0 +1,11 @@
+﻿@{
+    @ViewBag?.Data
+    @ViewBag.IntIndexer?[0]
+    @ViewBag.StrIndexer?["key"]
+    @ViewBag?.Method(Value?[23]?.More)(SecondValue)?["key"]
+}
+
+@ViewBag?.Data
+@ViewBag.IntIndexer?[0]
+@ViewBag.StrIndexer?["key"]
+@ViewBag?.Method(Value?[23]?.More)(SecondValue)?["key"]
```

Comment:
This is just combining everything in one, will remove the nested delegate part of this.